package.json

```diff
@@ -65,10 +65,10 @@
     "@babel/plugin-proposal-private-methods": "^7.18.6",
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
     "@dotlottie/react-player": "^1.6.3",
-    "@expensify/react-native-live-markdown": "0.1.69",
+    "@expensify/react-native-live-markdown": "0.1.70",
     "@expo/metro-runtime": "~3.1.1",
     "@formatjs/intl-datetimeformat": "^6.10.0",
-    "@formatjs/intl-listformat": "^7.2.2",
+  "@formatjs/intl-listformat": "^7.2.2",
```

Comment:
What happened to the indentation here? 😄

Reply:
😮 , I'll fix this asap